### DISCUSSION TITLE
Can I see the user who added a URL in Archivebox?

### DISCUSSION THREAD

**Discussion by Paulie420** [ArchiveBox/ArchiveBox#1624] [Q&A]
I have a small community of retro computer nerds and have been adding users to my Archivebox so they can add URLs. They can also add Tags and whatnot...

I might just be missing it right in front of me, but can I see what user archived a URL??

**Comment by pirate** [comment]
Yes it's a new feature in >= v0.8.5. Before that we didn't save the user info.
